src/inmanta/db/versions/v1.py

```diff
@@ -0,0 +1,262 @@
+async def update(connection):
+    schema = """
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+DROP TABLE IF EXISTS project CASCADE;
```

Comment:
isn't this very risky to put in the update script that should only every run if the DB is empty?

Reply:
Drop statements are removed.